src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/AssetCacheEntry.cs

```diff
@@ -1,11 +1,75 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using System;
+using UnityEngine;
+
 namespace Microsoft.MixedReality.SpectatorView
 {
+    [Serializable]
+    internal class AssetId
+    {
+        public static AssetId Empty { get; } = new AssetId(System.Guid.Empty, -1);
+
+        public StringGuid Guid => guid;
+
+        [SerializeField]
+        private StringGuid guid;
+
+        public long FileIdentifier => fileIdentifier;

```

Comment:
>FileIdentifier [](start = 20, length = 14)

Can you add a comment about what "FileIdentifier" is?

It's possible that this location isn't the right place for the comment (i.e. it's somewhat abstract at this location) but I'm trying to get a little more clarity for a reader who sees an abstract number (i.e. file ID), then look at this code, and ideally there should be some explanation/direction that this point that guides them to what's up